cds_migrator_kit/videos/weblecture_migration/load/helpers.py

```diff
@@ -0,0 +1,246 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# CDS-Videos is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-Videos migration load module helper."""
+
+
+import os
+import re
+import shutil
+import tempfile
+from pathlib import Path
+
+from cds.modules.deposit.api import deposit_video_resolver
+from cds.modules.deposit.ext import _create_tags
+from cds.modules.flows.api import AVCFlowCeleryTasks
+from cds.modules.flows.models import FlowTaskMetadata, FlowTaskStatus
+from cds.modules.flows.tasks import (
+    ExtractMetadataTask,
+    ExtractFramesTask,
+    TranscodeVideoTask,
+)
+from cds.modules.opencast.tasks import _get_opencast_subformat_info
+from cds.modules.records.api import CDSVideosFilesIterator
+from celery import chain as celery_chain
+from flask import current_app
+from invenio_db import db
+from invenio_files_rest.models import (
+    Bucket,
+    FileInstance,
+    ObjectVersion,
+    ObjectVersionTag,
+    as_object_version,
+)
+from invenio_files_rest.storage import pyfs_storage_factory
+
+from cds_migrator_kit.errors import ManualImportRequired
+
+
+def copy_file_to_bucket(bucket_id, file_path):
+    """Create a FileInstance, move the video file to FileInstance storage, return the created object version."""
+    try:
+        video_bucket = Bucket.get(bucket_id)
+        video_name = os.path.basename(file_path)
+        file = FileInstance.create()
+
+        # Get the location for the file instance
+        default_location = video_bucket.location.uri
+        file_storage = pyfs_storage_factory(
+            fileinstance=file, default_location=default_location
+        )
+        fp = file_storage.open(mode="wb")
+        full_path = Path(fp.name.decode()).resolve()
+
+        # Copy file to storage.
+        shutil.copy2(file_path, full_path)
```

Comment:
I would check for safety if the destination path exists already, not to overwrite an existing file